sympy/matrices/matrices.py

```diff
@@ -176,6 +176,30 @@ def is_echelon(self):
     def rank(self, iszerofunc=_iszero, simplify=False):
         return _rank(self, iszerofunc=iszerofunc, simplify=simplify)
 
+    def rref_rhs(self, rhs, iszerofunc=_iszero, simplify=False, pivots=True,
+        normalize_last=True):
```

Comment:
In new methods I would prefer not to add all of these options. I regret that they are already there in the existing functions but we should not add them in new functions unless there is some significant reason.

Here `_iszero` should be handled by `DomainMatrix`. It is a bad idea to use `simplify`. The `normalize_last` parameter will not make any sense of `DomainMatrix` is being used. The `pivots` return value was always a mistake at least it should never have been included in end user API.

I would reduce this signature to:
```python
def rref_rhs(self, rhs):
```
and not return the pivots.

Reply:
done